Apps/Sandcastle/gallery/Atmosphere HSB Color.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```

Comment:
The Sandcastle name is fine as just `Atmosphere Color.html`. Some other things to add to this demo might be a fog enabled button (like http://localhost:8080/Apps/Sandcastle/index.html?src=Terrain.html&label=Showcases), and set a different initial view, such as a view like your screenshot.